app/src/main/java/com/sopt/androidstudy/presentation/sign/screens/BindingConversions.kt

```diff
@@ -0,0 +1,24 @@
+package com.sopt.androidstudy.presentation.sign.screens
+
+import android.view.View
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+
+object BindingConversions {
+
+
+    @BindingAdapter("visible")
+    @JvmStatic
+    fun isVisible(view: TextView, is_enable: Boolean) {
+        when (is_enable) {
+            true -> {
+                view.visibility = View.INVISIBLE
+
+            }
+            false -> {
+                view.visibility = View.VISIBLE
+            }
+        }
+    }
+
+}
```

Comment:
```suggestion
@BindingAdapter("visible")
fun isVisible(view: View, visible: Boolean) {
    view.visibility = if (visible) View.INVISIBLE else View.VISIBLE
}
```

1. object와 @JvmStatic으로 감싸주셨군요.. [구글 공식 문서](https://developer.android.com/topic/libraries/data-binding/binding-adapters)에서는 @BindingAdapter 어노테이션만 사용한 예제로 나와있어 고대로 변경해봤습니다!
<img width="495" alt="스크린샷 2022-04-26 오후 11 41 45" src="https://user-images.githubusercontent.com/48701368/165326201-94a975a4-c982-47f4-82ce-c8b4a2107912.png">

2. 텍스트 뷰 뿐만이 아니라 각종 뷰들의 visibility를 조절할 경우를 고려해서 파라미터 중 view의 타입을 View 로 변경해도 좋을 것 같아요~!
3. `is_enable`.. visibility를 조절할 거 잖아요? isVisible이 좋아보입니다! 그리고 변수는 전부 카멜케이스로 통일시키는 것이 어떠신지 ㅎㅎ 
4. 조건값이 true, false 뿐이라면 if-else 문이 더 간결할 것 같습니다!

Reply:
1. 메모 - 공식문서 다시보기
2. 다른 뷰들까지 고려할 수 있다는 걸 생각 안했네요! 감사합니다!
3. 저도 왜 자주 스네이크 케이스를 쓸까요,,, 뭔가 어디선가 영향을 받은듯
4. if-else로 삼항연산자(랑 비슷한 기능을 하는) 처리!

Reply:
왕!!! BindingAdapter까지,,!! 영진언니 코드리뷰 보니 더 간단히 할 수 있군요!! 좋은 정보입니다 :)